src/Components/SmartComponents/SystemsExposedTable/SystemsExposedTable.js

```diff
@@ -202,7 +202,7 @@ const SystemsExposedTable = (props) => {
                 bulkSelect ={ selectOptions && {
                     count: selectedHosts && selectedHosts.length,
                     items: selectOptions.items,
-                    isDisabled: items.meta.total_items === 0,
+                    isDisabled: items.meta.total_items === 0 && (!selectedHosts || selectedHosts.length === 0),
```

Comment:
`(selectedHosts && selectedHosts.length === 0)` 
is the same and a bit more readable then
`(!selectedHosts || selectedHosts.length === 0)`

Reply:
I agree with you, what you suggested is much cleaner. But ` (selectedHosts && selectedHosts.length === 0)` does not give expected result. Because selectedHosts is set to undefined initially as we have dependency with Inventory here. To get rid if this case we need to do more investigation and changes.

Reply:
Whoops, I misread that as short circuit, sorry.